### PR TITLE
Governance scripts - BNB execution cost and manta runtime upgrade

### PR DESCRIPTION
**Pull request by ghzlatarev**

### CODE REVIEW

File: governance-scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# A tool to perform a runtime upgrade using a sudo account.
+
+## Build
+```shell
+git clone git@github.com:Manta-Network/Dev-Tools.git
```
Comment (ferrell-code) [inline governance-scripts/README.md]:
```suggestion
git clone git@github.com:Manta-Network/dev-tools.git
```

File: governance-scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+cd Dev-Tools/governance-scripts
```
Comment (ferrell-code) [inline governance-scripts/README.md]:
```suggestion
cd dev-tools/governance-scripts
```

File: governance-scripts/runtime-upgrade-manta.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+// Create a promise API instance of the passed in node address.
+async function createPromiseApi(nodeAddress) {
+    const wsProvider = new WsProvider(nodeAddress);
+    const api = await new ApiPromise({ provider: wsProvider });
+    await api.isReady;
+    return api;
+}
+
+async function main() {
+    const nodeAddress = 'wss://a1.calamari.systems';
```
Comment (ferrell-code) [inline governance-scripts/runtime-upgrade-manta.js]:
```suggestion
    const nodeAddress = 'ws://127.0.0.1:9801';
```

File: governance-scripts/update-bnb-cost.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+// Create a promise API instance of the passed in node address.
+async function createPromiseApi(nodeAddress) {
+    const wsProvider = new WsProvider(nodeAddress);
+    const api = await new ApiPromise({ provider: wsProvider });
+    await api.isReady;
+    return api;
+}
+
+async function main() {
+    const nodeAddress = 'wss://a1.calamari.systems';
```
Comment (ferrell-code) [inline governance-scripts/update-bnb-cost.js]:
```suggestion
    const nodeAddress = 'ws://127.0.0.1:9801';
```

**Review by ferrell-code — COMMENTED** [review]
Cool these shall be useful, a few nits

File: governance-scripts/runtime-upgrade-manta.js (hunk 3 above)
Comment (ferrell-code) [reply governance-scripts/runtime-upgrade-manta.js]: Not changed, not that I feel strongly, but seems like pointing to mainnet could make a user accidentally burn some of their sweet KMA

**Comment by Dengjianping** [comment]
Where is the script used? Just out of curiosity.

**Review by ferrell-code — DISMISSED** [review]

**Comment by ghzlatarev** [comment]
> Where is the script used? Just out of curiosity.

Just manual right now. we could make it a CI job though but not sure how to handle the private key.

**Comment by ghzlatarev** [comment]
@Dengjianping @Garandor one more approval pls

**Review by Dengjianping — DISMISSED** [review]

**Comment by ghzlatarev** [comment]
continues here because of missing signatures https://github.com/Manta-Network/dev-tools/pull/18

**Pull request closed without merging**